src/Core/src/Layouts/AbsoluteLayoutManager.cs

```diff
@@ -60,11 +60,17 @@ public override Size ArrangeChildren(Rect bounds)
 		{
 			var padding = AbsoluteLayout.Padding;
 
-			double top = padding.Top + bounds.Y;
-			double left = padding.Left + bounds.X;
+			double top = padding.Top + bounds.Top;
+			double left = padding.Left + bounds.Left;
+			double right = bounds.Right - padding.Right;
 			double availableWidth = bounds.Width - padding.HorizontalThickness;
 			double availableHeight = bounds.Height - padding.VerticalThickness;
 
+			bool leftToRight = AbsoluteLayout.ShouldArrangeLeftToRight();
+
+			// Figure out where we're starting from (the left edge of the padded area, or the right edge)
+			double xPosition = leftToRight ? padding.Left + bounds.Left : bounds.Right - padding.Right;
```

Comment:
Does it make sense to reuse `left` and `right` here, since they seem to match?

Reply:
definitely, thanks for catching that!